.gitignore

```diff
@@ -6,6 +6,15 @@
 target
 work
 *.iml
-yet-another-docker-java/dependency-reduced-pom.xml
+.*.swp
+.swp
 /target
 
+yet-another-docker-java/dependency-reduced-pom.xml
+yet-another-docker-java/.classpath
+yet-another-docker-java/.project
+yet-another-docker-plugin/.classpath
+yet-another-docker-plugin/.project
+yet-another-docker-plugin/.settings/org.eclipse.jdt.core.prefs
+yet-another-docker-plugin/.settings/de.loskutov.anyedit.AnyEditTools.prefs
```

Comment:
Yes, these rules mostly ignore Eclipse files generated by Maven.  AnyEdit is a plugin I have installed in my workspace.  We could alternately ignore `*/.classpath`, `*/.project`, `*/.settings`, etc.